lib/minimax.rb

```diff
@@ -1,15 +1,23 @@
 require_relative './game_rules.rb'
 require_relative './moves.rb'
 require_relative './primitives.rb'
+require_relative './player.rb'
+require_relative './players/ai.rb'
+
 class Minimax
   include Moves
   include GameRules
   attr_accessor :game
 
   def move(game)
     @game = game
-    current_player = game.player2
-    opponent = game.player1
+    if @game.player1.instance_of? Player::Ai
```

Comment:
Does it matter if the player is AI or not? The behavior is to switch, no?

Reply:
@ssunday just to make sure I follow this - in a situation where the human is player1 and ai is player2, when ai runs the minimax algorithm, current_player will need to be player2 (Ai). where AI is player1 - and Ai runs the minimax algorithm, current_player will need to be player1. and thereafter, recursion will keep switching between the players. so thats why I have the initial check for player1 and player2. could we chat tomorrow? just want to make sure I am following this right.

Reply:
Player order shouldn't matter as long as they adhere to the same interface. Let's chat tomorrow if you have questions on this.

Reply:
@ssunday: @naush and I discussed this during our meeting today. I will work on this while implementing my next feature which is to allow AI to play against AI. Will merge the PR and continue to work on this.